client/components/social-buttons/google.js

```diff
@@ -101,6 +101,10 @@ class GoogleLoginButton extends Component {
 		} );
 
 		this.setState( { isDisabled: false } );
+
+		if ( this.props.launchServiceOnLoad ) {
+			this.client.requestCode();
```

Comment:
We can probably wrap this with `try/catch` so it doesn't register an error in the console. Additionally, an event (`calypso_social_button_launched_on_load_failure/success`) could be of some use.